bundles/org.openhab.binding.haywardomnilogic/README.md

```diff
@@ -0,0 +1,401 @@
+# <bindingName> Binding
+
+The Hayward Omnilogic binding integrates the Omnilogic pool controller using the Hayward API.
+
+The Hayward Omnilogic API interacts with Hayward's cloud server requiring a connection with the Internet for sending and receiving information.
+
+## Supported Things
+
+The table below lists the Nest binding thing types:
+
+| Things                                  | Description                                                                                                                                    | Thing Type     |
+|-----------------------------------------|------------------------------------------------------------------------------------------------------------------------------------------------|----------------|
+| Hayward OmniLogix Connection            | Connection to Hayward's Server                                                                                                                 | bridge         |
+| Backyard                                | Backyard                                                                                                                                       | backyard       |
+| Body of Water                           | Body of Water                                                                                                                                  | bow            |
+| Chlorinator                             | Chlorinator																																	   | chlorinator    |
+| Colorlogic Light                        | Colorlogic Light                                                                                                                               | colorlogic     |
+| Filter                                  | Filter/Pump control                                                                                                                            | filter         |
+| Heater Equipment                        | Actual heater (i.e. gas)                                                                                                                       | heater         |
+| Relay                                   | A Thermostat to control the various aspects of the house's HVAC system                                                                         | relay          |
+| Virtaul Heater                          | A Virtual Heater that can controls the heater equipment based on priority                                                                      | virtualHeater  |
+## Discovery
+
+The binding will automatically discover the Omnilogic pool things from the cloud server using your Hayward Omnilogic credentials.
+
+## Thing Configuration
+
+_Describe what is needed to manually configure a thing, either through the (Paper) UI or via a thing-file. This should be mainly about its mandatory and optional configuration parameters. A short example entry for a thing file can help!_
```

Comment:
All this text in italic here with `_` is the placeholder text from the skeleton script and should be removed. This is guideline text on what can be written here and should be replaced with actual text.